pages/index.js

```diff
@@ -1,7 +1,27 @@
-export default function HomePage() {
+import ArtPieces from "@/components/ArtPieces/ArtPieces";
+import useSWR from "swr";
+
+const URL = "https://example-apis.vercel.app/api/art";
+
+export default function HomePage({ fetcher }) {
+  const { data, error, isLoading } = useSWR(URL, fetcher);
+
+  if (!data) {
+    return null;
+  }
```

Comment:
you don't need this extra check.

The way useSWR work is:
if there is no error and the fetch is not loading anymore, the data will be available